service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadDraftOrdersAboutToSubmitControllerTest.java

```diff
@@ -261,7 +261,29 @@ void shouldRemoveTemporaryFields() {
             "showCMOsSentToJudge", "cmosSentToJudge", "cmoUploadType", "pastHearingsForCMO", "futureHearingsForCMO",
             "cmoHearingInfo", "showReplacementCMO", "previousCMO", "uploadedCaseManagementOrder", "replacementCMO",
             "cmoSupportingDocs", "cmoJudgeInfo", "cmoToSend", "hearingsForHearingOrderDrafts",
-            "currentHearingOrderDrafts", "hearingOrderDraftKind"
+            "currentHearingOrderDrafts", "hearingOrderDraftKind",
+            "cmoToSendTranslationRequirements",
+            "orderToSend0",
+            "orderToSendTranslationRequirements0",
+            "orderToSend1",
+            "orderToSendTranslationRequirements1",
+            "orderToSend2",
+            "orderToSendTranslationRequirements2",
+            "orderToSend3",
+            "orderToSendTranslationRequirements3",
+            "orderToSend4",
+            "orderToSendTranslationRequirements4",
+            "orderToSend5",
+            "orderToSendTranslationRequirements5",
+            "orderToSend6",
+            "orderToSendTranslationRequirements6",
+            "orderToSend7",
+            "orderToSendTranslationRequirements7",
+            "orderToSend8",
+            "orderToSendTranslationRequirements8",
+            "orderToSend9",
+            "orderToSendTranslationRequirements9",
+            "orderToSendOptionCount"
```

Comment:
Could shorten this by grabbing these fields using the `temporaryFields` function in **UploadDraftOrdersData.java**

Reply:
When testing, it's better to be explicit, otherwise you'd generate the test assertion with a concrete method. If done in that way we are doing something like assertThat(object.method()).isEqualTo(object.method()), and if you have a bug in your method, you won't find it.